internal/model/provider.go

```diff
@@ -0,0 +1,23 @@
+package model
+
+import (
+	"fmt"
+	"strings"
+)
+
+const (
+	// ProviderAWS is the cloud provider AWS.
+	ProviderAWS = "aws"
+	// ProviderAzure is the cloud provider Azure.
```

Comment:
Just stubbing this out for later? :)

Reply:
Meh, yeah I'll just remove this, especially since it's not meaningfully complete.